Sources/SourceControl/Repository.swift

```diff
@@ -73,7 +73,7 @@ public protocol RepositoryProvider {
     ///   - repository: The specifier of the repository to fetch.
     ///
     /// - Throws: If there is any error fetching the repository.
-    func fetch(repository: RepositorySpecifier, to path: AbsolutePath) throws
+    func fetch(repository: RepositorySpecifier, to path: AbsolutePath, update: Bool) throws
```

Comment:
Since the protocol is being changed, could we add an extension here with an implementation of the new protocol method to call the old one? Otherwise clients will immediately fail to compile.  Thanks.

Reply:
Forgot about your comment from 23 days ago. I think we can go without this boolean flag.